Developer-Guide.md

```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
 ### Build the debug image
 
 Follow the instructions in the [Build a rootfs image](#build-a-rootfs-image)
-section.
+section when using rootfs and for initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.
 
 ### Configure runtime for custom debug image
 
 Install the image:
 
+**Note**: The image has to be changed from `kata-containers.img` to `kata-containers-initrd.img` while using initrd.
```

Comment:
Minor rewording maybe, to avoid any confusion about renaming the image:

> **Note**: When using an initrd image, use the image name `kata-containers-initrd.img` below, in place of the rootfs image name `kata-containers.img`.